pages/education/about-gi-bill-benefits/how-to-use-benefits/test-fees.md

```diff
@@ -79,19 +89,25 @@ We will pay you back for the approved tests listed below:
 
 [See the complete list of tests and fees](https://inquiry.vba.va.gov/weamspub/buildSearchNE.do).
 
-**We won’t pay you back for these costs:**
+### We won’t pay you back for certain fees and costs
+
+**These include:**
+
 - Pretest costs (such as Kaplan tests)
 - Fees to get scores quickly
 - Optional items not required to take an approved test
 
-For tests taken before August 1, 2018, if we paid you back for testing under the Post-9/11 GI Bill, we charged you at least a month of entitlement pay. For national tests taken after August 1, 2018, we'll prorate the entitlement charges under the Post-9/11 GI Bill based on the actual amount of the fee charged for the test.
+### How we charge your benefits
+
+**For tests taken before August 1, 2018,** if we paid you back for testing under the Post-9/11 GI Bill, we charged you at least a month of entitlement pay. 
+
+**For tests taken after August 1, 2018,** we'll prorate the entitlement charges under the Post-9/11 GI Bill based on the actual amount of the fee charged for the test.
+
 </div>
 </li>
-</ul>
 
-### How do I get this benefit?
+## How do I get this benefit?
```

Comment:
This heading can't be inside an accordion list, it'll throw an aXe error and break the build. I did some testing and if we close the previous list, and start a new one with the heading outside the list, it should pass fine.

```markdown
</ul>

## How do I get this benefit?
<ul class="usa-accordion">
  ... two list item accordions, For Licensing and Certification, For National Testing
</ul>
```

---
![screen shot 2019-02-25 at 9 17 15 am](https://user-images.githubusercontent.com/934879/53351008-0809bc00-38e6-11e9-8005-cf2a5df1b205.png)

Reply:
I think I fixed it. Take a look! And thanks for noticing this.